assets/js/constants/constants.ts

```diff
@@ -49,7 +49,7 @@ export const SCREEN_TYPES: { label: string; ids: string[] }[] = [
   { label: "E-Ink: Green Line", ids: ["gl_eink_v2"] },
   { label: "Elevator", ids: ["elevator_v2"] },
   { label: "PA ESS", ids: ["pa_ess"] },
-  { label: "Pre Fare Duo", ids: ["pre_fare_v2"] },
+  { label: "Pre Fare", ids: ["pre_fare_v2"] },
```

Comment:
nit: we pretty consistently refer to these as "Pre-Fare" screens (with hyphen) in other contexts, so it might make sense to use that here too.